src/Sulu/Bundle/ContactBundle/Api/Account.php

```diff
@@ -1088,12 +1103,24 @@ public function removeMedia(MediaInterface $media)
     /**
      * Get medias.
      *
+     * @deprecated Please use getMedia() instead.
+     *
+     * @return Media[]
+     */
+    public function getMedias()
+    {
+        return $this->getMedia();
```

Comment:
Sadly just the `deprecated` comment above is not enough to also trigger a deprecation in Symfony... In order for that to work you have to trigger an error in here:

```php
@trigger_error('The Account::getMedias method is deprecated, use Account::getMedia instead', E_USER_DEPRECATED);
```